pf/tfbridge/provider_diagnostics.go

```diff
@@ -37,7 +38,16 @@ func (p *provider) processDiagnostics(diagnostics []*tfprotov6.Diagnostic) error
 			if d.Attribute != nil {
 				prefix = fmt.Sprintf("[%s] ", d.Attribute.String())
 			}
-			return fmt.Errorf("%s%s: %s", prefix, d.Summary, d.Detail)
+			summary, summaryErr := tfbridge.ReplaceConfigProperties(d.Summary, p.info.Config, p.info.P.Schema())
```

Comment:
I'm curious why this is applied at procesDiagnostics, which is applying to *everything*, Configure, Create, you name it. This is a very wide net to cast. That may be the right thing to do but I'm curious is this intentional? For SDKv2 based providers the change seems a lot more surgical, only affecting Configure.

Reply:
Yeah, good point. Thanks for pointing that out - it was not intentional, should now be corrected.

Could be the right thing to do for other methods but it might be better to make the change once we have a concrete reason.